### PR TITLE
v5.13.1

### PR DESCRIPTION
**Pull request by m4theushw**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,54 @@
...
+#### Changes
+
+- [pickers] Add Italian (it-IT) locale (#5467) @felixh10r
+- [pickers] Add Spanish (es-ES) locale (#5468) @felixh10r
+- [pickers] Improve French (fr-FR) locale (#5446) @Zenoo
+- [pickers] Improve German (de-DE) locale (#5447) @sebastianfrey
+
+### Docs
+
+- [docs] FIx typo in the sorting page (#5431) @JosephMarinier
```
Comment (flaviendelangle) [inline CHANGELOG.md:41]:
```suggestion
- [docs] Fix typo in the sorting page (#5431) @JosephMarinier
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,54 @@
...
+- [docs] Fix description for "Striped rows" example (#5432) @lindapaiste
+- [docs] Make a clear difference between the license and license key (#5316) @oliviertassinari
+- [docs] Update pickers README files (#5456) @cherniavskii
+
```
Comment (joserodolfofreitas) [inline CHANGELOG.md:45]: Just merged this one.

```suggestion
[docs] Clarify the scope of support for MUI X (#5423) @joserodolfofreitas 

```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,54 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.13.1
+
+_Jul 14, 2022_
+
+We'd like to offer a big thanks to the 13 contributors who made this release possible. Here are some highlights ✨:
+
+- ✏️ Enter edit mode when pasting in a cell with Ctrl+V (#5405) @alexfauquette
```
Comment (joserodolfofreitas) [inline CHANGELOG.md:12]:
```suggestion
- ✏️ Enter edit mode when pasting in a cell with Ctrl+V (command+V on macs) (#5405) @alexfauquette
```

Comment (m4theushw) [reply CHANGELOG.md:12]:
```suggestion
- ✏️ Enter edit mode when pasting in a cell with Ctrl+V (Cmd+V on macOS) (#5405) @alexfauquette
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,55 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.13.1
+
+_Jul 14, 2022_
```
Comment (m4theushw) [inline CHANGELOG.md:8]:
```suggestion
_Jul 15, 2022_
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,55 @@
...
+- [pickers] Improve French (fr-FR) locale (#5446) @Zenoo
+- [pickers] Improve German (de-DE) locale (#5447) @sebastianfrey
+
+### Docs
+
+- [docs] Fix typo in the sorting page (#5431) @JosephMarinier
+- [docs] Fix description for "Striped rows" example (#5432) @lindapaiste
+- [docs] Make a clear difference between the license and license key (#5316) @oliviertassinari
+- [docs] Update pickers README files (#5456) @cherniavskii
+- [docs] Clarify the scope of support for MUI X (#5423) @joserodolfofreitas
```
Comment (m4theushw) [inline CHANGELOG.md:45]:
```suggestion
- [docs] Clarify the scope of support for MUI X (#5423) @joserodolfofreitas
```

**Review by joserodolfofreitas — APPROVED** [review]

**Comment by cherniavskii** [comment]
I've merged master into it to include #5403

**Review by cherniavskii — APPROVED** [review]

**Pull request merged**